modules/library/referencing/src/test/java/org/geotools/referencing/operation/projection/GeostationarySatelliteTest.java

```diff
@@ -69,6 +83,10 @@ public class GeostationarySatelliteTest {
     static MathTransform ellipsoidalGeosToGeog;
     static MathTransform geogToEllipsoidalGeos;
 
+    static CoordinateReferenceSystem esriGeosCRS;
+    static MathTransform esriGeosToGeog;
+    static MathTransform geogToEsriGeos;
```

Comment:
These two fields do not seem to be used (unlike the similar fields above, used in transformation tests). Do you plan to make use of them, or should they be removed?